wiki/Installing-Oppia-Android.md

```diff
@@ -81,36 +84,40 @@ Please follow these steps to set up Oppia Android on your local machine.
 ## Set up and run tests
 Testing the app is an integral part of our development process. You will need to test all code changes to ensure that the app works correctly, therefore it is important to ensure that your test configuration works.
 
-We run tests in either Espresso(`app` module tests) or Robolectric(non-app module tests), meaning tests will run in either the emulator or on Gradle/Bazel via the terminal.
+Robolectric allows for faster unit testing of Android applications without the need for device or emulator setup.
+
+### Configure Robolectric Tests
+These are tests that are in non-app modules, such as **domain** or **utility**.
```

Comment:
This needs to be modified now.

Reply:
Done.

Reply:
I believe this should be a subheading the same way you had "Step-by-Step guidance for setting up and running app modules robolectric test:" above, you can also make this bold, and someting like 
"Step-by-Step guidance for setting up and running non-app modules robolectric test:
These are tests that are in other  modules, such as **domain** or **utility**.

Also add some spacing so it is clear that it is a different section.

Reply:
I think you may have misunderstood what II meant here, I have added a suggestion as to what I meant.

Reply:
Basically, I needed it to be clear that we usually have different modules and that the test initialization will be different. The instructions are correct, but the section subtitle was not clear for non-app module tests.